php/src/Container/ContainerState.php

```diff
@@ -2,11 +2,35 @@
 
 namespace AIO\Container;
 
-enum ContainerState: string {
-    case ImageDoesNotExist = 'image_does_not_exist';
-    case NotRestarting = 'not_restarting';
-    case Restarting = 'restarting';
-    case Running = 'running';
```

Comment:
Sorry, I think I oversaw this. I think we need the Running state as otherwise the logic gets hard to understand, no?

Reply:
See my comment